.evergreen/install.sh

```diff
@@ -15,28 +15,13 @@ usage() {
 [ $# -lt 1 ] && { usage; exit 2; }
 
 VERSION=${1}
-PREFIX=${PREFIX:-$(pwd)"/deps-install"}
```

Comment:
Same with this file, a straight copy to build the latest C driver (the 1.9 version of libmongoc still used autotools to build)